target/linux/ath79/dts/qca9563_tplink_archer-c6-v2-us.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9563_tplink_archer-x6-v2.dtsi"
+
+/ {
+	compatible = "tplink,archer-c6-v2-us", "qca,qca9563";
+	model = "TP-Link Archer A6 v2 (US/TW) / C6 v2 (US)";
```

Comment:
I'd invert order here, as the first name should match the compatible/file name:
model = "TP-Link Archer C6 v2 (US) / A6 v2 (US/TW)";